Makefile

```diff
@@ -23,6 +23,10 @@ broker: $(SOURCES) ## Build the broker
 build: broker ## Build binary from source
 	@echo > /dev/null
 
+generate: ## regenerate mocks
+	go get github.com/vektra/mockery/.../
+	@go generate ./...
```

Comment:
I hate so many of the go tools, they totally don't work with symlinks. UGH! The minute I move to a real directory, it works.

Reply:
I will fix this after this PR gets merged. Looks like there might be a workaround to work with symlinks
https://github.com/vektra/mockery/issues/157#issuecomment-363510344